src/extension.ts

```diff
@@ -23,15 +23,17 @@ import {
   install,
   configurePaths,
   uninstall,
-  updateCLI,
+  //updateCLI,
   cleanup
 } from "./one-click/install";
 import { TextDecoder, TextEncoder } from "util";
+import { Logger } from "./logger";
 let analytics: Analytics;
 
 export var system: string;
 export const output = vscode.window.createOutputChannel("PROS Output");
-
+export var OneClickLogger: Logger;
```

Comment:
nit: class names should be `PascalCase`, but instances of classes should be `camelCase`